### ISSUE TITLE
Expo notifications inconsistency between EAS and Classic build.

### ISSUE THREAD

**Issue by jurica-penjgusic** [expo/expo#12722]
### Summary

I have Ios and Android app deployed with Classic build on stores.
When switched on EAS build services expo notifications are complaining about missing experience id.

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

managed

### What platform(s) does this occur on?

Android, iOS

### SDK Version (managed workflow only)

41

### Environment

Expo CLI 4.4.1 environment info:
System:
OS: Windows 10 10.0.19041
Binaries:
Node: 12.18.3 - C:\Program Files\nodejs\node.EXE
Yarn: 1.22.4 - C:\Users\Jurica\AppData\Roaming\npm\yarn.CMD
npm: 6.14.6 - C:\Program Files\nodejs\npm.CMD
IDEs:
Android Studio: Version 4.1.0.0 AI-201.8743.12.41.6953283
npmPackages:
expo: ^41.0.0 => 41.0.0
react: 16.13.1 => 16.13.1
react-dom: 16.13.1 => 16.13.1
react-native: https://github.com/expo/react-native/archive/sdk-41.0.0.tar.gz => 0.63.2
react-native-web: ~0.13.12 => 0.13.18
Expo Workflow: managed

### Reproducible demo or steps to reproduce from a blank project

 const token = await Notifications.getExpoPushTokenAsync();

**Comment by cruzach** [comment]
This should have been addressed by https://github.com/expo/expo/commit/7b8b8fabb369d136b047fbc27588a4b78b3f0205#diff-c149530cad2ce06cabbf7298905665502e29aaf359b5d83c7d02e299303472d3 and https://github.com/expo/expo/commit/5ed90708d2c39e4752755c8be27aaed325b0b213#diff-a44d72ea231e3ac9780f9ab5f22c86f85d828aeea1a7bbb15d6e2a01509ec836. What version of `expo-notifications` are you using, and what version of `expo-constants`?

**Comment by jurica-penjgusic** [comment]
```
 "expo-notifications": "~0.11.5",
  "expo-constants": "~10.1.3",
```

**Comment by cruzach** [comment]
Same version (`expo-notifications@0.11.6`) is working for me, so please provide a minimal reproducible demo

**Comment by jurica-penjgusic** [comment]
I have latest expo-cli (4.4.3) and expo install expo-notifications are giving me

`
"expo-notifications": "~0.11.5",
  "expo-constants": "~10.1.3",
`

**Comment by cruzach** [comment]
Like I said above, please provide a minimal reproducible demo

**Comment by jurica-penjgusic** [comment]
If you are not noticing we don't have same expo-notifications!
I can't install expo-notifications because expo install gives me ~0.11.5 and you have ~0.11.6

**Comment by cruzach** [comment]
~0.11.5 resolves to 0.11.6 - https://stackabuse.com/caret-vs-tilde-in-package-json/

**Comment by jurica-penjgusic** [comment]
Deleting yarn.lock, node modules then upgrading to 0.11.6 solves issues on both platforms!
Tnx anyway!

BTW EAS build is awesome!!
From 71 mb to 9 mb on IOS and 17 to 10 mb on android!

Keep up with good work!
